experimental/dds/tree2/src/test/snapshots/files/concurrent-inserts-tree3.json

```diff
@@ -342,44 +342,48 @@
                 "content": [
                   [
                     "rootFieldKey",
-                    [
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "z"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "x"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "d"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "f"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "e"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "y"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "a"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "b"
-                      },
-                      {
-                        "type": "com.fluidframework.leaf.string",
-                        "value": "c"
-                      }
-                    ]
+                    {
+                      "version": 1,
+                      "identifiers": [],
+                      "shapes": [
+                        {
+                          "c": {
+                            "type": "com.fluidframework.leaf.string",
+                            "value": true,
+                            "fields": []
+                          }
+                        },
+                        {
+                          "a": 2
+                        },
+                        {
+                          "d": 0
+                        }
+                      ],
+                      "data": [
```

Comment:
This looks like the compressed format, and the compression isn't doing as good of job as it should here. Its compressing this as a sequence of any nodes when the schema in these tests enforces that all these children should have the same schema so it should compress better. Not an issue in your PR probably, but something we should fix regardless.

Reply:
Added TODO in `schemaCompressedEncode` to use `FieldSchema` to improve the inefficient encoding.